src/platforms/index.mdx

```diff
@@ -41,4 +41,4 @@ These SDKs are maintained and supported by [the Sentry community](https://forum.
 
 Some platforms have been phased out or have been replaced with new SDKs. For those legacy integrations head over to the [legacy clients](/clients/) page.
 
-Your favorite language or framework still cannot be found? Then we encourage you to start a discussion about supporting it on our [community forum](https://forum.sentry.io).
+If you end up making a new integration, thank you. Let us know about it [in a PR](https://github.com/getsentry/sentry-docs/compare).
```

Comment:
Generally curious, *is* this (opening a PR in `sentry-docs`) the behavior we want for folks who build new integrations?

Reply:
Actually we _do_ have these docs. 😁

Updated in f39cb5f4679925f9ebb463f025a89eab671a7ca9. 👍

Reply:
I was about to say, lol